pkg/doc/did/schema.go

```diff
@@ -44,6 +44,14 @@ const (
     "id": {
       "type": "string"
     },
+    "alsoKnownAs": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "format": "uri"
```

Comment:
careful with the `uri` format.... test cases need to exist to cover this... make sure you are covering all of:

https://www.w3.org/TR/did-core/#also-known-as